diff --git a/src/hb-draw.h b/src/hb-draw.h
index 021078455..603ad0428 100644
--- a/src/hb-draw.h
+++ b/src/hb-draw.h
@@ -41,7 +41,7 @@ HB_BEGIN_DECLS
  * @path_start_y: Y component of the start of current path
  * @current_x: X component of current point
  * @current_y: Y component of current point
- * @slant_xy: Slanting factor for synthetic oblique, Since: 11.0.0
+ * @slant_xy: Slanting factor for synthetic oblique
  *
  * Current drawing state.
  *
